### ISSUE TITLE
New deployment does not run due to Postgres change

### ISSUE THREAD

**Issue by alhirzel** [singularityhub/sregistry#277]
**Describe the bug**

The following appears in `docker-compose logs`:

```
db_1         | Error: Database is uninitialized and superuser password is not specified.
db_1         |        You must specify POSTGRES_PASSWORD to a non-empty value for the
db_1         |        superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".
db_1         | 
db_1         |        You may also use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all
db_1         |        connections without a password. This is *not* recommended.
db_1         | 
db_1         |        See PostgreSQL documentation about "trust":
db_1         |        https://www.postgresql.org/docs/current/auth-trust.html
```

**To Reproduce**

Attempt `docker-compose up`.

**Comment by alhirzel** [comment]
Here's info on the workaround I have used: https://github.com/docker-library/postgres/issues/681#issuecomment-586811176

**Comment by vsoch** [comment]
This looks like we just need to pin the version of postgres. For now until there is more detailed developer time to look at the workaround you suggest, can you test `9.6.16`and if that works we can pin it in the docker-compose?

**Comment by vsoch** [comment]
Will `POSTGRES_HOST_AUTH_METHOD: trust` work for older versions as well?

**Comment by vsoch** [comment]
@alhirzel I'll have  PR for you to look at in about a minute that uses the envar.

**Comment by vsoch** [comment]
@alhirzel please test https://github.com/singularityhub/sregistry/pull/278. Thank you!

**Comment by alhirzel** [comment]
#278 works great

**Comment by vsoch** [comment]
okay let's get it deployed and built asap then! Thanks for catching this!

**Comment by vsoch** [comment]
Fixed with #278. @alhirzel you can use the branch, and wait for the containers to pop up at https://quay.io/repository/vanessa/sregistry?tab=tags
